unit_test/oci_container_test.py

```diff
@@ -63,13 +78,19 @@ def test_debug_info(container_engine):
 
 
 def test_environment(container_engine):
-    with OCIContainer(engine=container_engine, image=DEFAULT_IMAGE) as container:
+    with modified_os_environ(CIBUILDWHEEL="1", SOURCE_DATE_EPOCH="42"), OCIContainer(
+        engine=container_engine, image=DEFAULT_IMAGE
+    ) as container:
```

Comment:
There's a feature in pytest for this - [monkeypatch.setenv](https://docs.pytest.org/en/7.1.x/how-to/monkeypatch.html). It sets the env var for the duration of the test.

Reply:
Thanks for the tip, I'm not a pytest user.